aws-janitor/resources/route53.go

```diff
@@ -177,7 +179,7 @@ func (rrs Route53ResourceRecordSets) MarkAndSweep(opts Options, set *Set) error
 			}
 
 			if _, err := svc.ChangeResourceRecordSets(deleteReq); err != nil {
-				logger.Warningf("unable to delete DNS records: %v", err)
+				logger.Warningf("unable to delete DNS records for zone %v with error: %v", zone.Id, err)
```

Comment:
I think this is going to log the pointer, you probably want `aws.StringValue(zone.Id)`